### PR TITLE
[plugin.video.popcorntimes] v2.0

### PR DESCRIPTION
**Pull request by V10lator**
### Description
Video plugin for popcorntimes.tv
### Checklist:
- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

**Comment by V10lator** [comment]
A bit more information:

- **General**
Add-on name: Popcorntimes.tv
Add-on ID: plugin.video.popcorntimes
Version number: 2.0
Kodi/repository version: matrix

- **Code location**
URL: https://github.com/V10lator/plugin.video.popcorntimes

File: plugin.video.popcorntimes/addon.xml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  <provides>video</provides>
+</extension>
+<extension point="xbmc.addon.metadata">
+  <summary lang="en_GB">Video plugin for popcorntimes.tv</summary>
+  <description lang="en_GB">Video plugin for popcorntimes.tv</description>
+  <disclaimer lang="en_GB">Video plugin for popcorntimes.tv</disclaimer>
+  <summary lang="de_DE">Video plugin für popcorntimes.tv</summary>
+  <description lang="de_DE">Video plugin für popcorntimes.tv</description>
+  <disclaimer lang="de_DE">Video plugin für popcorntimes.tv</disclaimer>
+  <license>GNU GENERAL PUBLIC LICENSE. Version 2</license>
```
Comment (enen92) [inline plugin.video.popcorntimes/addon.xml]: please use a spdx compliant identifier for the license: GPL-2.0-only or GPL-2.0-or-later

File: plugin.video.popcorntimes/main.py [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+
+def play(movie_url):
+	title = movie_url
+	liz = xbmcgui.ListItem( '')
+	streamUrl = getStream(movie_url)
+	if streamUrl: 
+		fullurl = streamUrl.strip() + "|Referer=" + movie_url.strip()
+		liz.setPath(fullurl)
+		xbmcplugin.setResolvedUrl(_handle, True, liz)
+	#else:
```
Comment (enen92) [inline plugin.video.popcorntimes/main.py:158]: you might want to error out here, with a xbmcgui.Dialog().ok for example - not mandatory

**Review by enen92 — CHANGES_REQUESTED** [review]
minors, see comments.
Thanks for the contribution

File: plugin.video.popcorntimes/addon.xml (hunk 1 above)
Comment (V10lator) [reply plugin.video.popcorntimes/addon.xml]: Thanks for the heads up, I changed it.

File: plugin.video.popcorntimes/main.py (hunk 2 above)
Comment (V10lator) [reply plugin.video.popcorntimes/main.py:158]: Good point, will keep it in mind for a later version.

**Comment by V10lator** [comment]
Changed the license identifier as requested.

**Review by enen92 — APPROVED** [review]

**Pull request merged**